util/workers.py

```diff
@@ -26,13 +28,16 @@ def get_worker_count(worker_kind_name, multiplier, minimum=None, maximum=None):
     minimum = minimum or multiplier
     maximum = maximum or (multiplier * multiplier)
 
-    # Check for an override via an environment variable.
+    # Check for a process-specific override via an environment variable.
     override_value = os.environ.get("WORKER_COUNT_" + worker_kind_name.upper())
-    if override_value is not None:
+    if override_value:
```

Comment:
This should still check against `None`

Reply:
Why against None?

Reply:
To be consistent with how the other checks are done.
And in this case its generally better, since if for some reason the value of override_value was 0, it would still evaluate correctly.

Reply:
That is a good point -- if the environment variable is set, and somehow ends up with a value which evaluates to False, then we should handle that case accordingly.

Fixed.